Tasks/SonarQubePostTest/PRCA/ReportProcessor-Module.psm1

```diff
@@ -1,6 +1,8 @@
-$ProjectGuidAndFilePathMap = @{}
-$ComponentKeyAndPathMap = @{}
-$ComponentKeyAndRelativePathCache = @{}
+$script:ProjectGuidAndFilePathMap = @{}
```

Comment:
As far as I can tell, the script scope will prevent these variables from being accessed from outside the module.